automatic/intunewinapputil/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Win32 Content Prep Tool
+
+      Manage Windows Apps (.intunewin) with Intune
+
+      [See release notes for more information.](https://github.com/Microsoft/Microsoft-Win32-Content-Prep-Tool/releases)
+
+      Use the Microsoft Win32 Content Prep Tool to pre-process Windows Classic apps. The packaging tool converts application installation files into the .intunewin format. The packaging tool also detects the parameters required by Intune to determine the application installation state. After you use this tool on your apps, you will be able to upload and assign the apps in the Microsoft Intune console.
+
+      Before you install and the use Microsoft Win32 Content Prep Tool you **must**:
```

Comment:
Please change this line and everything beneath it to be inside of a `Notes` section.

Reply:
I presume you mean line 9 and below. The display from GitHub is a little confusing and makes it seem like maybe line 7, but I believe it's usually the last line displayed that the comment is regarding.